scripts/run_jmeter_benchmark_for_jenkins_api.sh

```diff
@@ -1,12 +1,8 @@
-#!/bin/bash
+#!/bin/sh
 
-# The PWD is the root of the project, so create the path to the jmeter test file
-jmeter_file_path="$PWD/gumby/experiments/tribler/api_benchmark/$TEST_RUNNER_FILE"
-
-# jmeter has no output directory option, so CD into this directory and run it in there.
-pushd $OUTPUT_DIR
+# jmeter has no output directory option, so CD into this directory and run it in there
+cd ${OUTPUT_DIR}
 
 # Run jmeter headless (-n option) and specify the file using -t
-jmeter -n -t $jmeter_file_path
+jmeter -n -t ${EXPERIMENT_DIR}
```

Comment:
I think you're thinking about something else. I mean the `.jmx` file a.k.a the test plan for jmeter